tests/integration/actions/settings/__init__.py

```diff
@@ -0,0 +1 @@
+"""Init for settings tests."""
```

Comment:
A package `__init__`'s docstring should describe the whole package, not `__init__.py` itself.
```suggestion
"""Settings tests."""
```

Reply:
done in next commit